documentation/devon4j.asciidoc

```diff
@@ -165,7 +169,7 @@ include::guide-batch-layer.asciidoc[leveloffset=+3]
 
 === Guides
 
-include::guide-configuration.asciidoc[leveloffset=+3]
+include::spring/guide-spring-configuration.asciidoc[leveloffset=+3]
```

Comment:
I think @hohwille wanted to have the spring stuff left where it is and not in a subfolder to have the links to the files stay as they are.

Reply:
I did what he described in this comment https://github.com/devonfw/devon4j/pull/435#discussion_r688476019, so we have a configuration guide with general stuff and then link from there to the framework-specific guides

Reply:
jep, wanted to keep the existing asciidocs and therefore links stable. New guides that are entirely spring specific should go to `spring` folder.